src/services/email.service.js

```
@@ -0,0 +1,56 @@
+import 'dotenv/config';
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
```

Consider adding `secure` or `tls` options to the transporter configuration and handling missing env values more robustly (clear error/log if SMTP_HOST/SMTP_USER/SMTP_PASSWORD are missing). This helps diagnostics and prevents silent failures in production/dev.